.github/workflows/build-2.0.yml

```diff
@@ -4,10 +4,27 @@ on:
   push:
     branches:
       - "2.0"
+  pull_request:
+    branches:
+      - "2.0"
 
 jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Test with pytest
+        if: always()
+        env:
+          VIADOT_CONFIG_PATH: .config/credentials.json.template
+        run: |
+          pip install pytest
+          sudo apt install libsqliteodbc
+          pytest tests/unit
+
   publish-docker-image:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/dev' && github.event_name == 'push'
```

Comment:
This is wrong, we're not dealing with the `dev` branch here

Reply:
Changed it to 2.0

Reply:
This action already only applies to 2.0 so this line is not needed